vnext/Microsoft.ReactNative/Views/FrameworkElementViewManager.cpp

```diff
@@ -448,6 +449,9 @@ bool FrameworkElementViewManager::UpdateProperty(
           element, states[static_cast<int32_t>(winrt::Microsoft::ReactNative::AccessibilityStates::Expanded)]);
       DynamicAutomationProperties::SetAccessibilityStateCollapsed(
           element, states[static_cast<int32_t>(winrt::Microsoft::ReactNative::AccessibilityStates::Collapsed)]);
+      if (auto peer = xaml::Automation::Peers::FrameworkElementAutomationPeer::FromElement(element)) {
+        peer.RaiseAutomationEvent(xaml::Automation::Peers::AutomationEvents::LiveRegionChanged);
```

Comment:
```cpp
xaml::Automation::Peers::AutomationEvents::LiveRegionChanged
```
> Identifies the event that is raised when a live region changes.

I'm a bit fuzzy on a11y, but know `accessibilityLiveRegion` is a separate prop on the RN side from `accessibilityState`. Is this the right event to be firing?

Reply:
accessibilityLiveRegion wires directly to [AutomationLiveSetting](https://docs.microsoft.com/en-us/dotnet/api/system.windows.automation.automationlivesetting?view=net-5.0) which specifies if/how the screen reader should be notified about changes to the components. From my understanding, LiveRegionChanged automation event is how we notify narrator of those changes.

Reply:
I wouldn't recommend using LiveRegion here. LiveRegion is used when you explicitly want Narrator to say something and in the majority of cases, it bypasses all heuristics that Narrator has to decide what to announce (based on the verbosity level specified by the user). Having too many events like that can degrade the experience for the user.

Can you check if the toggle change events are actually getting fired? I wonder if there is an issue there.